honeybee_radiance/cli/rpict.py

```diff
@@ -0,0 +1,74 @@
+"""honeybee radiance rpict command."""
+import click
+import sys
+import logging
+import os
+
+from honeybee_radiance.config import folders
+from honeybee_radiance_command.rpict import Rpict, RpictOptions
+
+
+_logger = logging.getLogger(__name__)
+
+
+@click.group(help='Commands to run rpict in Radiance.')
+def rpict():
+    pass
+
+
+@rpict.command('rpict')
+@click.argument(
+    'octree', type=click.Path(exists=True, file_okay=True, resolve_path=True)
+)
+@click.argument(
+    'view', type=click.Path(exists=True, file_okay=True, resolve_path=True)
+)
+@click.option(
+    '--rad-params', show_default=True, help='Radiance parameters.'
+)
+@click.option(
+    '--rad-params-locked', show_default=True, help='Protected Radiance parameters. '
+    'These values will overwrite user input rad parameters.'
+)
+@click.option(
+    '--output', '-o', show_default=True, help='Path to output file. If a relative path'
+    ' is provided it should be relative to project folder.'
```

Comment:
You have `show_default` here but there's no default value. Also, we should agree on a standard for what we call this type of option since I have been calling this `--output-file` in at least 50 other commands [like this one](https://github.com/ladybug-tools/honeybee-energy/blob/master/honeybee_energy/cli/translate.py#L113-L115)

Reply:
This is a tricky one - I'm all about making the libraries consistent. If we want to make this change we have to make it in other `honeybee-radiance` commands too like this one: https://github.com/ladybug-tools/honeybee-radiance/blob/c14f20e6ec3003e22a55d3667e9fe408f8290c01/honeybee_radiance/cli/raytrace.py#L33-L36

I think we might want to make the change from `--output` to `--output-file` all together in a separate commit. We have to update the plugins + recipes accordingly. But I'm all for it!